deploy/chart/docs/guide/rancher.md

```
@@ -0,0 +1,162 @@
+test-1-harbor-core-754c57dd77-xqfkb         1/1     Running   0             61s
+test-1-harbor-exporter-867746db74-5pgmn     1/1     Running   0             61s
+test-1-harbor-jobservice-67bc99bd86-7bvlp   0/1     Running   3 (35s ago)   61s
+test-1-harbor-portal-57f7894ff7-hfs8h       1/1     Running   0             60s
+test-1-harbor-registry-5bd8d59648-9fqs5     1/1     Running   0             61s
+test-1-harbor-scanner-trivy-0               1/1     Running   0             61s
+test-1-valkey-59486f6977-nqb2z              1/1     Running   0             61s
+```
+
+Since RKE2 comes with an Nginx ingress deployed by default, we can access the portal from a browser at:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)